framework/src/Volo.Abp.TextTemplating.Abstractions/Volo/Abp/TextTemplating/ITemplateRendererProvider.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections.Generic;
+using System.Globalization;
+using System.Threading.Tasks;
+using JetBrains.Annotations;
+
+namespace Volo.Abp.TextTemplating
+{
+    public interface ITemplateRendererProvider
```

Comment:
Consider to rename as `ITemplateRenderingEngine` following my previous suggestion.